conf/inventory/rhel-7.7-server-x86_64.yaml

```diff
@@ -1,6 +1,7 @@
 version_id: 7.7
 id: rhel
 instance:
+  enable-eus: True
```

Comment:
As we are getting the option via the CLI arguments, do we need to have this option in this file?

Reply:
i have renamed this option to `eus-supported` . add this option in the inventory file will let us determine if the EUS is supported for this distro or not. so if any body accidently uses `--enable-eus` for other distros where this option is not available, the test fails and saves lot of time instead of failing at the subscription enable code